docs/node/mainnet/eden-upgrade.md

```diff
@@ -214,15 +214,15 @@ Ubuntu versions (or other distributions using older system libraries).
 [systemd]: https://systemd.io/
 [Supervisor]: http://supervisord.org/
 [Wipe state]: ../run-your-node/maintenance/wiping-node-state.md#state-wipe-and-keep-node-identity
-[23.0.3]: https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.3
+[23.0.x]: https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.x
```

Comment:
This link needs to point to an actual release, can use the latest one, currently 23.0.4.

Reply:
I agree, @amela. Let's avoid writing 23.0.x and just replace 23.0.3 with 23.0.4 (and 23.0.5, if we make another release before the upgrade) except in my next comment below.